lib/fba_tools/fba_toolsImpl.pm

```diff
@@ -3,9 +3,9 @@ use strict;
 use Bio::KBase::Exceptions;
 # Use Semantic Versioning (2.0.0-rc.1)
 # http://semver.org 
-our $VERSION = '1.7.6';
+our $VERSION = '1.7.8';
```

Comment:
Can you bump the version in kbase.yml and add to the release notes in the README.md?